p4-16/spec/P4-16-spec.adoc

```diff
@@ -2765,7 +2765,7 @@ either by the number of arguments or by the names of the arguments,
 when calls are specifying argument names.  Argument type information
 is not used in disambiguating calls.
 
-Notice that overloading of parsers, controls, or packages is not allowed:
+Notice that overloading of abstract methods, parsers, controls, or packages is not allowed:
```

Comment:
I would just point out this forbids overloading of abstract methods completely, i.e. also by the number of arguments. I am OK with that, but I wanted to highlight that.

Reply:
Yes, it was intended because allowing overload via only the arity, and only for abstract methods, would be too complex. But in some future, if someone finds such a feature useful, we might consider lift this restriction then.